rxnetty/src/main/java/io/reactivex/netty/protocol/http/UnicastContentSubject.java

```diff
@@ -138,7 +137,14 @@ private UnicastContentSubject(final State<T> state, long noSubscriptionTimeout,
      */
     public boolean disposeIfNotSubscribed() {
         if (state.casState(State.STATES.UNSUBSCRIBED, State.STATES.DISPOSED)) {
-            state.bufferedSubject.subscribe(Subscribers.empty()); // Drain all items so that ByteBuf gets released.
+            state.bufferedSubject.subscribe(new Action1<T>() {
+                @Override
+                public void call(T t) {
+                    if (t instanceof ReferenceCounted) {
```

Comment:
I think we should release it just once as we only retain once. So, you can simply use `ReferenceCountUtil.release(Object msg)` method.